packages/analytics/analytics-chart/src/composables/useExploreResultToTimeDatasets.ts

```diff
@@ -176,6 +176,7 @@ export default function useExploreResultToTimeDataset(
             total: filled.reduce((acc, { y }) => acc + Number(y), 0),
             ...defaultLineOptions,
             fill,
+            borderWidth: borderWidth && !fill, // only show chart line if single dimension, and not "filled"
```

Comment:
The rule should be show border if not filled. Dimensionality should not be used to determine it